src/framework/mlt.vers

```diff
@@ -552,3 +552,17 @@ MLT_6.20.0 {
     mlt_profile_scale_height;
     mlt_properties_set_string;
 } MLT_6.18.0;
+
+MLT_6.22.0 {
+  global:
+    mlt_audio_set_values;
+    mlt_audio_get_values;
```

Comment:
In addition to these batch values set/get functions, I we need discrete functions as well. If not now, surely eventually. It would suck to have to change a value, not have all of them, and have to get existing values before setting them.